lambdas/src/utils/TimeRefreshedDataHolder.ts

```diff
@@ -0,0 +1,24 @@
+export class TimeRefreshedDataHolder<T> {
+  private value: T;
+  private valuePromise: Promise<T>;
+
+  constructor(private readonly provider: () => Promise<T>, private readonly refreshTime: number) {}
```

Comment:
Could we make `refreshTime` a string, and just call `ms(refreshTime)` internally?